### ISSUE TITLE
Display two title bars on Linux

### ISSUE THREAD

**Issue by zimuyanghua** [kikipoulet/SukiUI#123]
Using sukiwindow does not display properly on linux
> <suki:SukiWindow xmlns="https://github.com/avaloniaui"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:vm="using:sukiDemo.ViewModels"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:suki="clr-namespace:SukiUI.Controls;assembly=SukiUI"
        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
        x:Class="sukiDemo.Views.MainWindow"
        x:DataType="vm:MainWindowViewModel"
        Title="sukiDemo">
![image](https://github.com/kikipoulet/SukiUI/assets/46125819/b42c6e9c-b912-4a23-919e-a13704e5acbf)

When I modify to the following code, linux shows fine but windows cannot maximize the display

> <suki:SukiWindow xmlns="https://github.com/avaloniaui"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:vm="using:sukiDemo.ViewModels"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:suki="clr-namespace:SukiUI.Controls;assembly=SukiUI"
        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
        x:Class="sukiDemo.Views.MainWindow"
        x:DataType="vm:MainWindowViewModel"
        **SystemDecorations="BorderOnly"**
        Title="sukiDemo">
![image](https://github.com/kikipoulet/SukiUI/assets/46125819/2d6f451c-4bf1-47d2-be7f-2d579bb9c6e2)

`

**Comment by mugongliu1** [comment]
Try This Below：

TransparencyLevelHint="Transparent"
ExtendClientAreaToDecorationsHint="True"
SystemDecorations="None"

**Comment by zimuyanghua** [comment]
> Try This Below：
> 
> TransparencyLevelHint="Transparent" ExtendClientAreaToDecorationsHint="True" SystemDecorations="None"

Still no, double-clicking the title bar and button maximize doesn't go full screen, but dragging the border actually does go full screen
https://github.com/kikipoulet/SukiUI/assets/46125819/a960015e-b248-4f15-9e44-49132e58ce60

**Comment by kikipoulet** [comment]
Unfortunately it is more an Avalonia behavior. I know there are some issues handling linux with wayland as window manager.

> > Try This Below：
> > TransparencyLevelHint="Transparent" ExtendClientAreaToDecorationsHint="True" SystemDecorations="None"
> 
> Still no, double-clicking the title bar and button maximize doesn't go full screen, but dragging the border actually does go full screen https://github.com/kikipoulet/SukiUI/assets/46125819/a960015e-b248-4f15-9e44-49132e58ce60

I think this is the best you'll be able to get

**Comment by zimuyanghua** [comment]
@kikipoulet Thanks.
Although avalonia does not have full screen if SystemDecorations is set to BorderOnly, avalonia native windows do not have a double title bar in linux.
![image](https://github.com/kikipoulet/SukiUI/assets/46125819/b18c18ec-c393-4941-a43e-f610c8ba206b)

I can only take a compromise
```
if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
{
      this.SystemDecorations = SystemDecorations.BorderOnly;
}
```

**Comment by kikipoulet** [comment]
It is a good idea, but it can be difficult to deal with the different linux windows manager. Wayland may need that, but others don't. Anyway, it can't be bad to at least have a better support for wayland, you can make a PR if you want or I will do it later.

**Comment by kikipoulet** [comment]
Fixed in commit 668de66473589e0cf1c4eadc52da195cf5fd12fb
